src/test/java/io/swagger/oas/test/examples/ExampleBuilderTest.java

```diff
@@ -780,4 +780,22 @@ public void testRefAndInlineAllOf(@Injectable final List<AuthorizationValue> aut
         Assert.assertNotNull(openAPI.getComponents().getSchemas().get("User"));
         Assert.assertTrue(openAPI.getPaths().get("/refToAllOf").getGet().getResponses().get("200").getContent().get("application/json").getSchema().getProperties().size() == 2);
     }
+
+    @Test
+    public void schemaExampleWithNullProperty(@Injectable final List<AuthorizationValue> auths) throws Exception {
+        ParseOptions options = new ParseOptions();
+        options.setResolve(true);
+        options.setResolveFully(true);
+        OpenAPI openAPI = new OpenAPIV3Parser().read("src/test/swagger/schemaExampleWithNullProperty.yaml",auths,options);
+
+        Assert.assertNotNull(openAPI);
+        ApiResponse apiResponse = openAPI.getPaths().get("/schemaExampleWithNullProperty").getGet().getResponses().get("200");
+        Example example = ExampleBuilder.fromSchema(apiResponse.getContent().get("application/json").getSchema(),null,ExampleBuilder.RequestType.READ);
+        String output = Json.pretty(example);
```

Comment:
@webron Regarding your first point. It is not the ExampleBuilder which converts it to JSON. The `application/json` is just a name without meaning here. I convert it to JSON in the test to easily compare it. I could as well have used `Yaml.pretty(example)` to output YAML instead.

Reply:
Thanks. The main issue is that I'm not sure what the expected behavior should be. Stay tuned.